dem/core/registry.py

```diff
@@ -3,26 +3,24 @@
 """
 # dem/core/registry.py
 
-import json
-import docker
-import subprocess
-
+import requests
 from dem.core import container_engine as container_engine
+from dem.core.exceptions import RegistryError
 
 def list_repos(container_engine_obj: container_engine.ContainerEngine) -> list[str]:
-    registryimagelist = []
+    registry = "axemsolutions"
     images = []
```

Comment:
Is that file part of the feature?

Reply:
No, but I had to merge DEM-55 into this branch, so the tests didn't break. 
After merging #28, this diff won't show up.